project.yaml

```diff
@@ -1333,10 +1333,13 @@ Layer:
                       AND admin_level = '2'
                       AND way_area > 9e+12
                       AND ST_Within(ST_Centroid(l.way), p.way)
+                      AND p.way && !bbox!
```

Comment:
I was actually wrong about this line. It has a marginal impact, so I'll remove it to simplify.